Client/Frontend/Settings/AutoplaySettingsViewController.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import Foundation
+import Shared
+
+class AutoplaySettingsViewController: SettingsTableViewController {
+    /* variables for checkmark settings */
+    let prefs: Prefs
+    var currentChoice: AutoplayAction!
+    init(prefs: Prefs) {
+        self.prefs = prefs
+        super.init(style: .grouped)
+
+        self.title = .Settings.Autoplay.Autoplay
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func generateSettings() -> [SettingSection] {
+        self.currentChoice = prefs.stringForKey(AutoplayAccessors.AutoplayPrefKey).flatMap({AutoplayAction(rawValue: $0)}) ?? AutoplayAccessors.Default
+
+        let onFinished = {
+            self.prefs.setString(self.currentChoice.rawValue, forKey: AutoplayAccessors.AutoplayPrefKey)
+            self.tableView.reloadData()
+        }
+
+        let allowAudioAndVideo = CheckmarkSetting(
+            title: NSAttributedString(string: .Settings.Autoplay.AllowAudioAndVideo),
+            subtitle: nil,
+            accessibilityIdentifier: "AllowAudioAndVideo",
```

Comment:
For a11y identifiers, we define the identifiers in the `AccessibilityIdentifiers.swift` file, in a related section.

Reply:
ah that looks new. i could only find two other places where the AccessibilityIdentifiers class was being used. done.